FinanceServicesApi/V1/Controllers/LeaseholdAssetSearchController.cs

```diff
@@ -40,18 +35,21 @@ public LeaseholdAssetSearchController(IGetLeaseholdAssetsListUseCase getLeasehol
         public async Task<IActionResult> GetLeaseholdAssetsByAddress([FromQuery] LeaseholdAssetsRequest housingSearchRequest)
```

Comment:
The request should contain the from year and we need to give response until the current year

Reply:
Done